src/helpers/makeQuery.js

```diff
@@ -0,0 +1,85 @@
+// @flow
+import { ErrorWithStack } from './errors';
+
+const prepareErrorMessage = error =>
```

Comment:
This is copied and pasted. Import this thing instead.

Reply:
Fixed that for you